### ISSUE TITLE
[KBFS] No way to abort upload

### ISSUE THREAD

**Issue by mbloms** [keybase/client#21858]
I accidentally copied alot of files to KBFS. It hogs my internet and prevents me from uploading files I actually want to upload. I've tried removing the folders from KBFS. Restarting Windows and even disabling and reenableling KBFS in explorer. There seems to be no way to stop Keybase from uploading these files.

How can I stop uploads in progress?

**Comment by mbloms** [comment]
Now it seems to be uploading things into the void

```
C:\Users\Mikael>keybase fs rm -r /keybase/private/jennifer327,mbloms
- ERROR Cannot recursively delete /keybase/private/jennifer327,mbloms

C:\Users\Mikael>keybase fs rm -r /keybase/private/mbloms,jennifer327
- ERROR Cannot recursively delete /keybase/private/jennifer327,mbloms

C:\Users\Mikael>keybase fs uploads
/keybase/private//Inspelning-006.aac
/keybase/private//Inspelning-007.aac
/keybase/private//Inspelning-009.aac
/keybase/private//Inspelning-010.aac
/keybase/private//Inspelning-011.aac
/keybase/private//Inspelning-012.aac
/keybase/private//VID_20191215_170152.mp4
/keybase/private//VID_20191215_170415.mp4
/keybase/private/jennifer327,mbloms
/keybase/private/jennifer327,mbloms/Lucia2019
/keybase/private/jennifer327,mbloms/Lucia2019/IMG_20191215_165343.jpg
/keybase/private/jennifer327,mbloms/Lucia2019/Inspelning-008.aac
/keybase/private/jennifer327,mbloms/Lucia2019/Inspelning-013.aac

1.20 GB left to upload
Estimated time remaining: 10h59m21s
```

**Comment by mbloms** [comment]
I finally just gave up on this and deprovisioned the computer and logged in again. That did the trick, but I wouldn't call it a user friendly experience.

**Comment by srikanthops** [comment]
I have the same problem.. a important feature is missing here.. please 
I cannot deprovision my laptop

**Comment by mrfgl** [comment]
> Disabling and Enabling the Finder Sync fixes it for me. Not a permanent solution but gets the job done.

_MacOS Catalina 10.15.7_

**Settings** > **Files** > **Disable Finder Integration**

_then_

**Settings** > **Files** > **Enable Finder Integration**

> Disabling and Enabling the Finder Sync fixes it for me. Not a permanent solution but gets the job done.

**Comment by mattborja** [comment]
Yeah, this is really unacceptable. At least give us a way to cancel the operation in progress.

I accidentally did the same thing myself on a laptop because of contact with the touchpad, causing something very large to inadvertently get picked up and dropped onto the desktop app.

Basically, I had to:

1. Quit Keybase completely (via File/app menu) to stop the bleed
2. Download and install the mobile app
3. Delete the files from the mobile app
4. Track down the queue for the desktop app and manually purge, or uninstall the app altogether and remove data
